sys-auth/AusweisApp2/AusweisApp2-9999.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake git-r3
+
+DESCRIPTION="Official authentication app for German ID cards and residence permits"
+HOMEPAGE="https://www.ausweisapp.bund.de/"
+EGIT_REPO_URI="https://github.com/Governikus/AusweisApp2.git"
+
+LICENSE="EUPL-1.2"
+SLOT="0"
+KEYWORDS=""
+IUSE=""
+
+BDEPEND="virtual/pkgconfig"
+
+RDEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtconcurrent:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtgui:5
+	dev-qt/qtsvg:5
+	dev-qt/qtwebsockets:5[qml]
+	dev-qt/qtquickcontrols2:5
```

Comment:
I don't see this depdency used? It that linked with your local build?

Reply:
It could be used in qml imports, which would make it an RDEPEND only.

Reply:
I will look later in the source code. Haven't yet digged deep into it.

Reply:
Now moved to DEPEND (see 1.20.0 conversation below)